pb/bytestring.go

```diff
@@ -0,0 +1,43 @@
+package dht_pb
+
+import (
+	"encoding/json"
+)
+
+type byteString string
+
+func (b byteString) Marshal() ([]byte, error) {
+	return []byte(b), nil
+}
+
+func (b *byteString) MarshalTo(data []byte) (int, error) {
+	// intentionally using KeyString here to avoid allocating.
```

Comment:
KeyString?

Reply:
Sorry, copypasta from the cid version of this.